src/middleware/blocks.test.js

```diff
@@ -67,4 +67,23 @@ describe('middleware/blocks', () => {
     expect(dispatchedAction.payload).toHaveProperty('blockNumbers', expectedBlockNumbers);
     expect(dispatchedAction.payload).toHaveProperty('blocks', expectedBlocks);
   });
+
+  it('should dispatch an error if the request fails', async () => {
+    mockAction = {
+      type: t.FETCH_BLOCKS,
+      payload: {
+        requestedBlockNumber: 20,
+        amountOfBlocks: 4,
+      },
+    };
+
+    mockAdapter.getBlocks = jest.fn(() => new Promise((resolve, reject) => reject([{}, { ok: false }])));
```

Comment:
Since you don't actually do something with the reject value you could leave it at `=> reject()`